coding-concepts.js

```diff
@@ -6,54 +6,60 @@
 
+// b) Verify and explain: The output was 10! It's important to note it was 10 as a NUMBER and not a string, because it's performing a mathematical operation, so the returned value will be a number.
 
 // --------------------2) What will this log?
 
-const greeting = "Hello World!"
-// console.log(greeting[4])
+const greeting = "Hello World!";
+// console.log(greeting[4]);
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I believe this code is checking to access the 4th index of an Array, but "Hello World!" is not an Array, so it will most likely return "undefined".
 
+// b) Verify and explain: I was wrong! The output was "o". I didn't realize that the command [4] would check for the index of a character even outside of an array, because I previously thought that the method .indexOf() was the only way to do that. Good to know!
 
 // --------------------3) What will this log?
 
-const languages = ["JavaScript", "Ruby", "Python", "C++"]
-const index = 1
-// console.log(languages[index])
+const languages = ["JavaScript", "Ruby", "Python", "C++"];
+const index = 1;
+// console.log(languages[index]);
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I believe this will log the string "Ruby" to the console, because it's using the number 1 (set as the value for the variable index), and that value will be passed into the brackets to return what is located in index 1 of the array. It should be the same as:
+// console.log(languages[1]).
 
+// b) Verify and explain: I was correct! Even though we're passing a variable name into the array call, it's still going to use the value of that array, which is the number 1.
 
 // --------------------4) What will this log?
 
-const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+const weekendDays = ["saturday", "sunday"];
+// console.log(weekendDays.toUpperCase());
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I'm not sure if this is going to work. Because the data is inside of an array, I believe you have to specify what element you want to apply the method to for it to work. Since this is applying the method to the ENTIRE array, I think it's going to either log the array as it is, or return an error.
+
+// b) Verify and explain: I was right! It returned the error "weekendDays.toUpperCase is not a function". I'm not 100% sure why it thinks that the method call is a function, but I've actually seen this error before while helping some of my classmates with a previous question. Just to test it out, I'm going to try the code:
+
+// console.log(weekendDays[0].toUpperCase());
+// output: "SATURDAY"
 
+// Yep! I was right, you have to specify what element you want to apply the method to. I would love to know why it thinks it's a function when you apply it to the whole array, though!
```

Comment:
Awesome work on this problem!  This error is a pretty common one that will often happen when you are using a built in method on a data type that it is not made to work with.  Ultimately a built in method is a type of a function (we will get more into this later this week).